### PR TITLE
New tip. String concatenation

### PR DESCRIPTION
**Pull request by gogainda**

### CODE REVIEW

**Comment by loverajoel** [comment]
Hey @gogainda take a look [here](https://developers.google.com/speed/articles/optimizing-javascript?csw=1&utm_campaign=Front%2BEnd%2BNewsletter&utm_medium=web&utm_source=Front_End_Newsletter_1)

**Comment by gogainda** [comment]
@loverajoel According to this benchmark 'concat' variant is faster than 'join' http://jsperf.com/string-concatenation-join-concat-plus

File: README.md [hunk 1]
```diff
@@ -18,6 +18,29 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+## #10 - Safe string concatenation
+
+> 2016-01-10 by [@gogainda]
+
+Suppose you have a couple of variables with unknown types and you want to concatenate them. To be sure that the arithmetical operation would not be applied during the concatenation:
+
+```javascript
+var one = 1;
+var two = 2;
+var three = "3"
```
Comment (loverajoel) [inline README.md]: `'3'` just commons conventions

File: README.md [hunk 2]
```diff
@@ -18,6 +18,29 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+
+var result = "".concat(one, two, three); //"123"
```
Comment (loverajoel) [inline README.md]: `''.concat`

File: README.md [hunk 3]
```diff
@@ -18,6 +18,29 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
 
 # Tips list
 
+## #15 - Safe string concatenation
+
+> 2016-01-15 by [@gogainda]
+
+Suppose you have a couple of variables with unknown types and you want to concatenate them. To be sure that the arithmetical operation would not be applied during the concatenation:
```
Comment (loverajoel) [inline README.md:24]: hey can you add a link to concat official documentation?

**Comment by loverajoel** [comment]
@gogainda awesome, can you add some about performance related? I think that will be more useful the tip.
Also squash your commits, please

File: README.md [hunk 4]
```diff
@@ -18,6 +18,33 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter.
 
 # Tips list
 
+## #18 - Safe string concatenation
+
+> 2016-01-18 by [@gogainda]
```
Comment (loverajoel) [inline README.md]: @gogainda can you update the date to today?

**Pull request merged**